### ISSUE TITLE
ArduinoQueue.h: No such file or directory

### ISSUE THREAD

**Issue by ghost** [EinarArnason/ArduinoQueue#16]
Arduino IDE v1.8.13
ArduinoQueue v1.2.3

```
$ cd ~
$ find . -name ArduinoQueue.h
./Arduino/libraries/ArduinoQueue/ArduinoQueue.h
```

Opened the example intQueueItemsSize and compiled:

`Queue.h: No such file or directory`

Why you want to include `Queue.h`?
Should not be: `#include ArduinoQueue.h"?

**Comment by laflaf3d** [comment]
same error, example is wrong

**Comment by EinarArnason** [comment]
Issue resolved in version 1.2.5
